src/librustdoc/html/static/rustdoc.css

```diff
@@ -1810,3 +1833,14 @@ details.rustdoc-toggle[open] > summary::before {
 	content: "[−]";
 	display: inline;
 }
+
+details.undocumented > summary::before {
+	content: "[+] Show hidden undocumented items";
```

Comment:
Interesting approach! This is different from how we've done the other toggles that have descriptions (`hide-me`), but it makes sense to do it this way, since this particular toggle has a description both in its open and in its closed state. 👍🏻